scripts/examples/typescript/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "xen-api-ts",
+  "version": "1.0.0",
+  "description": "A typescript library for Xen API",
```

Comment:
In fact it's not binndings either since we don't expose/bind any classes/fields/calls to the respective language concepts. Library or module sounds ok to me, but I agree that we can offer a bit more detail, like, as you said, that one can connect to XS and manage resources, pools, servers, VMs, SRs etc.